controller/internal/routes/identity_router.go

```diff
@@ -150,6 +150,18 @@ func (r *IdentityRouter) Register(ae *env.AppEnv) {
 		}, params.HTTPRequest, params.ID, "", permissions.IsAdmin())
 	})
 
```

Comment:
Why have explicit operations for these, instead of just a patch on the enabled/disabled flag? 

I don't think this is wrong, just curious as to why this is your preference

Reply:
The bool flag is virtual. It is derived from two other fields: `disabledAt` and `disabledUntil`. Identities can be locked indefinitely by not having `disabledUntil`  set.

Disabling is done by making a disable request with a duration or no duration (indefinite). Enabling is clearing both fields.

It is easier to define this behavior with a specific endpoint rather than updating two fields in concert.

Reply:
Makes sense, I just noticed the disabledUntil as I was further down.